server/midddleware/forwarder.go

```diff
@@ -86,9 +123,98 @@ func forwarderUnaryServerInterceptor() grpc.UnaryServerInterceptor {
 		txCtx := api.GetTransaction(ctx, req.(proto.Message))
 
 		if txCtx != nil && txCtx.GetOrigin() != types.MyOrigin {
-			return forwardRequest(ctx, txCtx, info.FullMethod, req.(proto.Message))
+			iface, err := forwardRequest(ctx, txCtx, info.FullMethod, req.(proto.Message))
+			log.Err(err).Str("method", info.FullMethod).Msg("forwarded request")
+			return iface, err
 		}
 
 		return handler(ctx, req)
 	}
 }
+
+func proxyHandler(_ interface{}, serverStream grpc.ServerStream) error {
+	fullMethodName, ok := grpc.MethodFromServerStream(serverStream)
+	if !ok {
+		return status.Errorf(codes.Internal, "lowLevelServerStream not exists in context")
```

Comment:
Don't we need to return tigris error from here?

Reply:
Let me change. But it actually doesn't matter. So as standard codes are mapped one to one to Tigris codes.